src/main/java/com/ably/kafka/connect/config/ConfigValueEvaluator.java

```diff
@@ -46,13 +46,17 @@ public Result evaluate(SinkRecord record, String pattern, boolean skippable) thr
         if (pattern == null) {
             return new Result(false, null);
         }
-        final byte[] key = (byte[]) record.key();
-        String keyString = null;
 
-        //we only want to evalutate UTF-8 encoded strings
-        if(key != null && ByteArrayUtils.isUTF8Encoded(key)) {
-            keyString = new String(key, StandardCharsets.UTF_8);
+        String keyString = null;
```

Comment:
We could split this section out into it's own method (can return null), just to split things up a bit?

Reply:
That's a good idea indeed. f627066282a18c564cf7088930fb14c8c85da858